specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -13221,6 +16065,111 @@
         }
       }
     },
+    "FunctionEnvelope": {
+      "description": "Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "FunctionEnvelope resource specific properties",
+          "properties": {
+            "name": {
```

Comment:
Not needed. Please remove

Reply:
The reason modifiable names are not allowed is that in this operation: https://github.com/EricSten-MSFT/azure-rest-api-specs/blob/449478bf179eae0001b9fd463355dd517e74fb6a/specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json#L2983 you could provide a different name here in the envelope from the one on the URL.

Reply:
A collection of FunctionEnvelope is returned in a response to ListFunctions operation.  In that case, the name is required to differentiate between functions.

Reply:
In ARM we use a read-only name for that purpose.

Reply:
Update #5 should have the fix.